config/archunit-store/slices_should_be_free_of_cycles_suppressions

```diff
@@ -799,7 +799,6 @@ Cycle detected: Slice api -> \
     - Method <com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocMissingLeadingAsteriskCheck.isLastLine(com.puppycrawl.tools.checkstyle.api.DetailNode)> calls method <com.puppycrawl.tools.checkstyle.api.DetailNode.getText()> in (JavadocMissingLeadingAsteriskCheck.java:202)\
     - Method <com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocParagraphCheck.visitJavadocToken(com.puppycrawl.tools.checkstyle.api.DetailNode)> calls method <com.puppycrawl.tools.checkstyle.api.DetailNode.getType()> in (JavadocParagraphCheck.java:202)\
     - Method <com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocParagraphCheck.visitJavadocToken(com.puppycrawl.tools.checkstyle.api.DetailNode)> calls method <com.puppycrawl.tools.checkstyle.api.DetailNode.getType()> in (JavadocParagraphCheck.java:203)\
-    - Constructor <com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocTypeCheck.<init>()> calls constructor <com.puppycrawl.tools.checkstyle.api.AbstractCheck.<init>()> in (JavadocTypeCheck.java:204)\
```

Comment:
@romani Why did this change on a simple document change?

Reply:
I am not sure, @HCY123902 , please confirm that this update is required by tool

Reply:
I think this is caused by the additional lines in Javadoc, which move the line positions of the constructors and methods in  `JavadocTypeCheck.java`. This change is automatically done by `mvn clean verify`. Without it, CI will complain